src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -102,7 +132,7 @@ function IOURequestStepAmount({
         );
     };
 
-    const navigateToNextPage = ({amount}: AmountParams) => {
+    const navigateToNextPage = ({amount, paymentMethod}: AmountParams) => {
```

Comment:
Shouldn't this whole thing be an action?

Reply:
I... don't think so? All the writing happens in an action

Reply:
Seems like we are fine! https://expensify.slack.com/archives/C03TQ48KC/p1713434903788039?thread_ts=1713359203.763509&cid=C03TQ48KC